src/TLuaInterpreter.cpp

```diff
@@ -174,7 +168,6 @@ void TLuaInterpreter::slot_replyFinished(QNetworkReply * reply )
     }
 
     QString localFileName = downloadMap.value( reply );
-<<<<<<< HEAD
     TEvent e;
```

Comment:
So we are going with actual event instances and passing by value - that is the safest way to do it and I will ensure that that is the paradigm that makes it through the `TEvent` instance I encounter in merging... :grin: